clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -40,8 +41,9 @@ public Builder(TachyonConf conf) {
       mBlockSizeBytes = conf.getBytes(Constants.USER_BLOCK_SIZE_BYTES_DEFAULT);
       mRecursive = false;
       mTTL = Constants.NO_TTL;
-      mUnderStorageType =
-          conf.getEnum(Constants.USER_FILE_UNDER_STORAGE_TYPE_DEFAULT, UnderStorageType.class);
+      WriteType defaultWriteType =
+          conf.getEnum(Constants.USER_FILE_WRITE_TYPE_DEFAULT, WriteType.class);
```

Comment:
This means the write type is configured once at deployment. In the future do we plan to allow write type configuration per call?